facts/4.2/oraclelinux-9-x86_64.facts

```diff
@@ -41,13 +42,16 @@
   "filesystems": "iso9660,xfs",
   "fips_enabled": false,
   "gem_version": "~> 4.2.0",
+  "hardwaremodel": "x86_64",
+  "hostname": "multi-1",
```

Comment:
Where did this come from?  For whatever historical reason, I thought the hostname is normally set to `foo` (and the domain to `example.com`). https://github.com/voxpupuli/facterdb/search?q=foo

Reply:
Oh... https://github.com/voxpupuli/facterdb/pull/219/files#diff-d078ad6295d8c9ba91465a8574948be08fbd0bedb0bfe84eeb46c1ee2d4fa8d2R46

Reply:
From the Vagrantfile. https://github.com/voxpupuli/facterdb/blob/master/facts/Vagrantfile#L9